packages/react/ds-core/package.json

```diff
@@ -22,7 +22,7 @@
     "build": "bun run build:package",
     "build:storybook": "storybook build",
     "build:package": "bun run build:package:tsc && bun run build:package:copycss",
-    "build:package:copycss": "copyfiles -u 1 src/ui/**/*.css dist/esm",
+    "build:package:copycss": "copyfiles -u 1 src/ui/**/**/*.css dist/esm",
```

Comment:
I had to add this because `Tooltip/common/TooltipArea/styles.css` was not being copied to dist on build, causing the boilerplate to fail to build.